src/library/recording/dlgrecording.cpp

```diff
@@ -107,8 +105,6 @@ DlgRecording::~DlgRecording() {
 }
 
 void DlgRecording::onShow() {
```

Comment:
...or delete them.

Reply:
still there because method in base class libraryview.h is pure virtual
https://github.com/mixxxdj/mixxx/blob/d0449aebd9744acedff3f59ebf6bec13bce0bbd0/src/library/libraryview.h#L18

I'll remove it from cpp.